app/util/files/FileUploadWindow.js

```diff
@@ -0,0 +1,70 @@
+// define a simple Window class to hold the grid
```

Comment:
As this is a view would it not make more sense to add it to app/view/fileuploader ?

Reply:
done